### PR TITLE
Fixes 646: kill Thanks page (and move it into a flash message of new issue)

### PR DESCRIPTION
**Pull request by miketaylr**
r? @magsout for CSS
r? @karlcow for Python

Not sure we want to merge just yet, it's sort of ugly. I'd like Mesha to do a UX review of the copy as well -- suggestions for words or design welcome from anyone!

(edit: also ignore that the header isn't styled -- that's taken care of in https://github.com/webcompat/webcompat.com/pull/1246)

screen shot 2016-12-16 at 5 37 07 pm

### CODE REVIEW

**Comment by miketaylr** [comment]
Oops:

```
FAIL: Test that /thanks/1 exists.
```

**Comment by adamopenweb** [comment]
Pinging @lockettm. Can you take a quick look at this please? Would love to have your feedback. :)

**Comment by miketaylr** [comment]
(also, the "Complain on Facebook" text is a joke)

**Comment by lockettm** [comment]
Copy looks good. 

I kind of like the "Complain on Facebook" text. :) If we want to add more context to the "Share on Twitter" and "Share on Facebook" buttons, a line above stating "Tell your friends about the bug you just filed," might work.

This could also be an opportunity to further engage users--explain what happens next and invite them to join the team. However, if the main goal is to get them to share on social media, this may split their attention.

**Comment by miketaylr** [comment]
Thanks for reviewing!

> This could also be an opportunity to further engage users--explain what happens next and invite them to join the team. However, if the main goal is to get them to share on social media, this may split their attention.

Yeah, I agree this is a good spot to recommend next steps and give suggestions on how they can help, etc. Let's file a bug to add that kind of stuff after we ship the simple version.

File: webcompat/static/css/development/components/flash-message.css [hunk 1]
```diff
@@ -29,3 +30,18 @@
...
     background-color: rgba(255, 89, 89, 0.92)
   }
+
+  .wc-FlashMessage.thanks {
+    border-style:solid;
+    border-width:1px;
+    border-color: #ffc900;
+    position: static;
+    width: var(--Issue-widthComment);
+    color: #fff;
```
Comment (magsout) [inline webcompat/static/css/development/components/flash-message.css]: This property is not required.

File: webcompat/static/css/development/components/flash-message.css [hunk 2]
```diff
@@ -29,3 +30,18 @@
     color: #fff;
     background-color: rgba(255, 89, 89, 0.92)
   }
+
+  .wc-FlashMessage.thanks {
```
Comment (magsout) [inline webcompat/static/css/development/components/flash-message.css]: `wc-FlashMessage--thanks` better than .thanks standalone.

File: webcompat/static/css/development/components/flash-message.css [hunk 3]
```diff
@@ -29,3 +30,18 @@
...
+    border-width:1px;
+    border-color: #ffc900;
+    position: static;
+    width: var(--Issue-widthComment);
+    color: #fff;
+    opacity: 1;
+    background-color: #fff;
+    color: #000;
+    text-align: left;
+    margin: 0;
```
Comment (magsout) [inline webcompat/static/css/development/components/flash-message.css]: `margin: 0 0 .8em 0;` added a space between flashmessage and issue; ;)

File: webcompat/static/css/development/components/flash-message.css (hunk 2 above)
Comment (miketaylr) [inline webcompat/static/css/development/components/flash-message.css]: done.

File: webcompat/static/css/development/components/flash-message.css (hunk 3 above)
Comment (miketaylr) [inline webcompat/static/css/development/components/flash-message.css]: done!

File: webcompat/static/css/development/components/flash-message.css (hunk 1 above)
Comment (miketaylr) [inline webcompat/static/css/development/components/flash-message.css]: done

**Comment by miketaylr** [comment]
screen shot 2016-12-22 at 11 24 38 am

OK, here we are. There's lots of room for improvement -- please file bugs and send PRs to make it better!

Thanks @karlcow @lockettm @magsout for review!

**Pull request merged**